subprojects/core/src/main/java/org/gradle/process/internal/worker/WorkerProcessBuilder.java

```diff
@@ -59,6 +59,10 @@ public interface WorkerProcessBuilder extends WorkerProcessSettings {
 
     void enableJvmMemoryInfoPublishing(boolean shouldPublish);
 
+    void useNativeServices(Boolean useNativeServices);
```

Comment:
Should be marked `@Nullable`. Same with the method that implements this

Reply:
I think a much better solution would be to use an enum with three values (`ENABLE`, `DEFAULT`, `DISABLE`), like we do in many places, e.g. with `WatchMode`:

https://github.com/gradle/gradle/blob/b8190353a5a319cb1a2af993bc76feb13f670bf7/platforms/core-execution/file-watching/src/main/java/org/gradle/internal/watch/registry/WatchMode.java#L22-L40

Reply:
Also works for me